### ISSUE TITLE
diplomat gem should be limitted to 2.0.0, not 2.0.1

### ISSUE THREAD

**Issue by EugenMayer** [markround/tiller#60]
2.0.1 does no longer work, it has a special k/v retrieval format

**Comment by EugenMayer** [comment]
see https://github.com/WeAreFarmGeek/diplomat/pull/120

**Comment by EugenMayer** [comment]
also see https://github.com/WeAreFarmGeek/diplomat/issues/152

**Comment by markround** [comment]
2.0.2 of that gem has been pushed apparently (according to https://github.com/WeAreFarmGeek/diplomat/issues/152) - does this fix the issue ?

**Comment by EugenMayer** [comment]
Yes, still consider making Diplomat a locked down version, if you ask me.

**Comment by markround** [comment]
https://github.com/markround/tiller/commit/21402eadd4f17b5a99cd95560025c48a49ac9ecd fixes this in the Gemfile. Currently in `develop` branch, it'll get rolled into the next release. I'll also add a note in the docs.

**Comment by markround** [comment]
Actually, scrap that. I'm going to have to do some more work for compatibility with v2.x of that Gem anyway. Watch this space...
